backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -234,6 +238,13 @@ public async Task<AvailabilitySlotDto> GetAvailabilitySlotById(long id)
 
             if (!updateAvailabilityDto.Schedule.WithTeamMembers)
             {
+                var scheduleId = availabilitySlot.SlotMembers.First().ScheduleId;
```

Comment:
Why first?

Reply:
Currently we can have only one slotmember

Reply:
But in the future we will have more that 1 and you need to think about it

Reply:
Fixed